sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ThroughputControlGroupConfigBuilder.java

```diff
@@ -176,8 +176,11 @@ public ThroughputControlGroupConfig build() {
         if (StringUtils.isEmpty(this.groupName)) {
             throw new IllegalArgumentException("Group name cannot be null nor empty");
         }
+        if (this.targetThroughput == null && this.targetThroughputThreshold == null && this.priorityLevel == null) {
+            throw new IllegalArgumentException("All targetThroughput, targetThroughputThreshold and priorityLevel cannot be null or empty.");
+        }
         if (this.targetThroughput == null && this.targetThroughputThreshold == null) {
-            throw new IllegalArgumentException("Neither targetThroughput nor targetThroughputThreshold is defined.");
+            this.targetThroughputThreshold = 1.0;
```

Comment:
I would like to see whether we can set it to a value >> 1 - why? In some scenarios backend actually allows for significantly more than 100% of the expected throughput - when no throughput control threshold is specified we should not limit at all on the clinet. So something like 100, 1000 or even int max value

Reply:
This is changed to int max on targetThroughput.